.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: android
+
+android:
+  components:
+    - build-tools-21.1.2
+    - android-19
+  licenses:
+    - android-sdk-license-5be876d5
+
+before_script:
+    - ./gradlew assemble
+
+script:
+    - ./gradlew check
+
+notifications:
+  email: false
```

Comment:
it's listed here: http://docs.travis-ci.com/user/notifications/

notifications:
  email:
    recipients:
      - one@example.com
      - other@example.com
    on_success: [always|never|change] # default: change
    on_failure: [always|never|change] # default: always